main.py

```diff
@@ -192,6 +194,13 @@ def run_commands(
                 print(f"Ignoring commands for {self.language}... ({c})")
             return None
 
+        system = platform.system().lower() # linux (wsl included), darwin (mac), windows
+        if self.machine_os and self.machine_os != system:
+            if config.debugging:
+                print(f"Skipping command since it is not for the current OS: {self.machine_os}, current: {system}")
+            return None
```

Comment:
manually tested this on a linux machine. So it's good. hard to test, can figure out later